.travis.yml

```diff
@@ -1,9 +1,18 @@
-sudo: false
+sudo: true
 services:
   - redis-server
 language: ruby
 rvm:
   - '2.3'
 before_install:
-  - cd ruby
   - gem install bundler -v 1.5.1
+  - cd python
+  - sudo pip install -U pip
+  - sudo pip install setuptools==33.1.1
+  - pip --version
+  - sudo pip install -r requirements.txt
+  - cd ../ruby
+
+after_script:
```

Comment:
Ya, I couldn't find a quick solution either. 
How do you want to deal with this going forward? Should we split this up into separate projects, or should we move towards a build system that supports both?